pkgs/by-name/co/composefs/package.nix

```diff
@@ -35,7 +35,12 @@ stdenv.mkDerivation (finalAttrs: {
   strictDeps = true;
   outputs = [ "out" "lib" "dev" ];
 
-  postPatch = lib.optionalString installExperimentalTools ''
+  postPatch =
+    ''
+      substituteInPlace libcomposefs/meson.build \
+        --replace-fail "both_libraries" "library"
```

Comment:
Turns out that `both_libraries` results in double the disk usage for the normal build (both static and shared libraries get compiled) as well as break static builds since it insists on building the shared library. So we disable it on our side.

Reply:
can you put this useful observation in a comment in the code?

Reply:
Will do